app/scripts/views/confirm_reset_password.js

```diff
@@ -139,21 +141,20 @@ define(function (require, exports, module) {
     },
 
     _finishPasswordResetSameBrowser (sessionInfo) {
-      // Only the account UID, unwrapBKey and keyFetchToken are passed
-      // from the verification tab. Load other from localStorage
-      var account = this.user.getAccountByUid(sessionInfo.uid);
-
+      const account = this.user.getAccountByUid(sessionInfo.uid);
+
+      // A bug in e10s causes localStorage in about:accounts and content tabs to be isolated from
+      // each other. Writes to localStorage from /complete_reset_password are not able to be read
+      // from within about:accounts. Because of this, all account data needed to sign in must
+      // be passed between windows. See https://github.com/mozilla/fxa-content-server/issues/4763
+      // and https://bugzilla.mozilla.org/show_bug.cgi?id=666724
+      //
       // keyFetchToken and unwrapBKey are sent from the verification tab,
```

Comment:
I wondered the same, as well as whether we are able to remove `keyFetchToken` and `unwrapBKey` from the list of sent fields since we no longer support Hello. 

We'll break reset password for fx_desktop_v1, and *possibly* fx_ios_v1 where the originating tab has to send the `login` message to the browser, but most users won't open the password reset link in Firefox for iOS anyways.

I can open an issue to explore, I didn't want to fix one thing to inadvertently break another. The comment should at a minimum be updated to mention fx_desktop_v1 and fx_ios_v1.